analysis_schema/SchemaModel.py

```diff
@@ -1,9 +1,7 @@
 from typing import List, Optional
 
-from pydantic import fields
-
-from analysis_schema.BaseModelFunctions import ytBaseModel
-from analysis_schema.ytDataClasses import Dataset, Visualizations
+from .BaseModelFunctions import ytBaseModel
```

Comment:
At some point we will want to rename these files to be things like `base_model_functions` but that's out of scope here

Reply:
That's a good idea. I'm going to fix the style issues and then merge.